ember-cli-build.js

```diff
@@ -26,6 +26,21 @@ module.exports = function(defaults) {
                 'node_modules/c3',
             ],
         },
+        inlineContent: {
+            raven: {
+                enabled: !LEAN_BUILD,
```

Comment:
Should this be `enabled: LEAN_BUILD`, so it's enabled in production?

Reply:
Ah yes, when I was testing locally, it the editor wasn't updating so I thought it was the other way around. You're correct though.